includes/acl/core/memory_utils.h

```diff
@@ -105,7 +105,24 @@ namespace acl
 			static inline DestIntegralType cast(SrcEnumType input)
 			{
 				typedef typename std::underlying_type<SrcEnumType>::type SrcIntegralType;
-				ACL_ENSURE(static_cast<SrcIntegralType>(input) >= std::numeric_limits<DestIntegralType>::min() && static_cast<SrcIntegralType>(input) <= std::numeric_limits<DestIntegralType>::max(), "static_cast would result in truncation");
+
+#if defined(ACL_USE_ERROR_CHECKS)
+				const SrcIntegralType integralInput = static_cast<SrcIntegralType>(input);
+
+				const bool srcSigned = std::numeric_limits<SrcIntegralType>::is_signed;
```

Comment:
coding standard says: src_signed, dest_signed, etc
Please change for consistency